src/NServiceBus.Core/Unicast/Transport/TransportConnectionString.cs

```diff
@@ -1,26 +1,40 @@
 ﻿namespace NServiceBus
 {
     using System;
-    using System.Configuration;
     using Transport;
 
     sealed class TransportConnectionString
     {
         TransportConnectionString()
         {
+#if NET452
+            GetValue = () => ReadConnectionStringFromAppConfig(DefaultConnectionStringName);
+#else
```

Comment:
not sure I like the else here. I think it would be better to stay explicit instead?